css/style.css

```diff
@@ -1,7 +1,17 @@
+html {
+    height: 100%;
+    overflow-y: visible;
+}
+
 #navbar {
     transition: 0.4s;
 }
 
+.divall {
+    overflow-x: hidden;
+    height: 100%;
+}
+
```

Comment:
Removed the scrollbar as it acted only on the single class and not on the complete window.